applications/ShapeOptimizationApplication/test_examples/01_Strain_Energy_Minimization_3D_Hook/optimization_parameters.json

```diff
@@ -1,18 +1,80 @@
+                "input_filename" : "3D_Hook"
+            },
+            "design_surface_sub_model_part_name" : "design_surface",
+            "damping" : {
+                "apply_damping" : true,
+                "damping_regions" : [{
+                        "sub_model_part_name"   : "support",
+                        "damp_X"                : true,
+                        "damp_Y"                : true,
+                        "damp_Z"                : true,
+                        "damping_function_type" : "cosine",
+                        "damping_radius"        : 85.0
+                    },{
+                        "sub_model_part_name"   : "inner_face",
+                        "damp_X"                : true,
+                        "damp_Y"                : true,
+                        "damp_Z"                : true,
+                        "damping_function_type" : "cosine",
+                        "damping_radius"        : 30.0
+                },{
+                        "sub_model_part_name"   : "tip_face",
+                        "damp_X"                : true,
+                        "damp_Y"                : true,
+                        "damp_Z"                : true,
+                        "damping_function_type" : "linear",
+                        "damping_radius"        : 20.0
+                }]
+            },
+            "mesh_motion" : {
+                "apply_mesh_solver" : true,
+                "solver_settings" : {
+                    "solver_type": "mesh_solver_structural_similarity",
+                    "model_part_name"    : "3D_Hook",
+                    "model_import_settings"              : {
+                        "input_type"     : "use_input_model_part"
+                    },
+                    "time_stepping" : {
+                        "time_step"       : 1.0
+                    },
+                    "domain_size"     : 3,
+                    "mesh_motion_linear_solver_settings"  : {
+                        "solver_type"         : "AMGCL",
+                        "max_iteration"       : 500,
+                        "tolerance"           : 1e-7,
+                        "provide_coordinates" : false,
+                        "smoother_type"       : "ilu0",
+                        "krylov_type"         : "bicgstab",
+                        "coarsening_type"     : "aggregation",
+                        "scaling"             : false,
+                        "verbosity"           : 1
+                    },
+                    "compute_reactions"     : false
+                },
+                "boundary_conditions_process_list" : [
+                ]
+            }
+        },
         "objectives" : [{
             "identifier" : "mass",
             "type"       : "minimization",
             "use_kratos" : true,
             "kratos_response_settings":{
                 "response_type"          : "mass",
+                "model_part_name"    : "3D_Hook",
```

Comment:
Why do we have to specify this model_part_name and model_import_settings, can't we use default values?

Reply:
Like for other responses (e.g. Strain energy), the name and also the import mode of the model part has to be given in the settings. For the mechanichal responses, this information is in the `primal_settings.json`, for the mass response function, we don't have an external settings file, thus giving the information directly here.